test/run-drun/max-stack-variant.mo

```diff
@@ -2,7 +2,7 @@
 import { errorMessage; debugPrint; } = "mo:⛔";
 
 actor {
-    let expectedMinimumSize = 26_000;
+    let expectedMinimumSize = 31_000;
```

Comment:
I just noticed that the version before this PR already supported 31_000. So, there seems to be another issue that the test does not scale to 62_000 with doubled stack size in EOP.

Reply:
I believe this is the Wasm stack overflow. So, probably, the limit 31_000 is also to be expected for this PR. (I changed the limit back to 31_000 like in `master`.)